src/main/resources/js/view.treant.js

```diff
@@ -185,6 +163,30 @@ function addTooltip() {
 	}
 }
 
+function addContextMenuToTreant() {
+	console.log("view.treant.js addContextMenuToTreant");
+	var treantNodes = document.getElementsByClassName("node");
+	var i;
+	for (i = 0; i < treantNodes.length; i++) {
+		treantNodes[i].addEventListener('contextmenu', function(event) {
+			event.preventDefault();
+			// TODO Find correct position in issue module
+			var left = event.pageX;
```

Comment:
@Jcl15 , @lw2011 , @dekome , @larstralle , does anybody know how to get the correct position for the context menu in the issue module? Please commit it on this branch, I would be very happy if you find out.